.github/workflows/ci.yml

```diff
@@ -98,18 +98,22 @@ jobs:
         # all the `!redeploy` comments, to get the next deployment number.
         # See https://docs.github.com/en/rest/deployments/deployments?apiVersion=2022-11-28#list-deployments
         run: |
+          # We --slurp the results because --paginate introduces potentially multiple array results
           pr_deployments=$(gh api \
-            -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" \
-            --paginate \
-            /repos/${{ github.repository }}/deployments \
-            --jq '[.[] | select(.ref == "${{ steps.pr.outputs.head }}")] | length'
+              -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" \
+              --paginate --slurp \
+              /repos/${{ github.repository }}/deployments \
+            | jq '[select(.[][].ref == "${{ steps.pr.outputs.head }}")] | length'
```

Comment:
I got an error running this on the command line
```bash
gh api -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" --paginate /repos/access-nri/access-test/deployments | jq 'select(.[][].ref = "paginate_deployments") | length'
```
error:
```
jq: error (at <stdin>:0): Cannot index string with string "ref"
```
But this works:
```bash
gh api -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" --paginate /repos/access-nri/access-test/deployments | jq '[select(.[].ref = "paginate_deployments")] | length'
```

Reply:
You need to add the `--slurp` option to the first command for it to work.

Reply:
Time for an update, then! :laughing: